sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -875,4 +931,9 @@ public void update(Observable observable) {
             suppressOrRevive();
         }
     }
+
+    @VisibleForTesting
+    void setApiServerDisabled(boolean disabled) {
```

Comment:
too much `DefaultSchedulerTest` infrastructure to rewrite.  You should take up scope expansion as a hobby.